src/js/components/PageHeader/stories/Simple.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+
+import {
+  Anchor,
+  Box,
+  Button,
+  Menu,
+  PageHeader,
+  Page,
+  PageContent,
+  ResponsiveContext,
+} from 'grommet';
+import { More } from 'grommet-icons';
+
+const actions = [
+  {
+    label: 'Primary Action',
+    primary: true,
+  },
+  {
+    label: 'Some Action',
+    secondary: true,
+  },
+  {
+    label: 'Other Action',
+    secondary: true,
+  },
+  {
+    label: 'Another Action',
+    secondary: true,
+  },
+];
+
+export const Simple = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Page>
+      <PageContent>
+        <PageHeader
+          title="Page title that is really long so we can test the wrapping 
+          behavior."
+          subtitle="The theme is setting the min-width for the columns at 
+          various breakpoints. This gives the caller control about what gets 
+          layout priority. In this case, we never want to let the 
+          title/subtitle get below a certain width."
+          actions={
+            size !== 'small' ? (
+              <Box direction="row" gap="small" align="center">
+                <Button label="Add Item" primary />
+                <Button label="Some Action" secondary />
+                <Button label="Other Action" secondary />
+                <Button label="One More Action" secondary />
+              </Box>
+            ) : (
+              <Menu items={actions} icon={<More />} />
+            )
+          }
+          context={<Anchor label="Parent Page" />}
```

Comment:
I know this naming can be changed in the theme just putting it out there if we can think of a better name for `context`.

Reply:
Agree on changing the name for this. We are using this section for navigation elements but the naming `context` doesn't give me any clue that this is the area for the up button or breadcrumbs.

Reply:
Good comments. Originally I had `contextualNav`. Would this be more clear?

Reply:
Good questions. I think `context` allows it to contain elements that provide "context" without them having to be navigational. For instance, it might make sense for a page about your car to have a title containing your license plate with a context of "Car", without having it be "Cars" as a navigational link. I suspect this example is on the 20% side though. But, I'm just trying to contemplate it through a wider lens of what someone might want to do with grommet. I could be OK with `contextNav`, as long as the caller can pass anything they want for it.